Serializer/BaseSerializerHandler.php

```diff
@@ -27,6 +27,11 @@
     protected $manager;
 
     /**
+     * @var array
+     */
+    private static $formats = array('json', 'xml', 'yml');
```

Comment:
Yeah but can you be sure the constructor is called before the call to the static method happens ?